docs/agents/chopper_mc2000b.rst

```diff
@@ -0,0 +1,56 @@
+.. highlight:: rst
+
+.. _chopper_mc2000b_agent:
+
+========================
+Chopper MC2000B Agent
```

Comment:
Can we rename this Agent to "Thorlabs MC2000B Agent"? That maintains consistency with most of our other Agent names in the form "<Manufacturer> <Model Number> Agent".